ext_localconf.php

```diff
@@ -13,4 +13,10 @@
         $GLOBALS['TYPO3_CONF_VARS']['SYS']['fluid']['namespaces']['fc'] = [];
     }
     $GLOBALS['TYPO3_CONF_VARS']['SYS']['fluid']['namespaces']['fc'][] = 'SMS\\FluidComponents\\ViewHelpers';
+
+    $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['fluid_components']['typeAliases'] = [
```

Comment:
Damit die Extension-Reihenfolge keine Rolle spielt, würde ich sowas bevorzugen:

```php
if (!isset($var)) {
  $var = [];
}
$var[...] = ...;
```

Zur Sicherheit und Konsistenz bitte die Klassennamen mit `\` beginnen lassen.

Reply:
oh yes, you are right